docs/ides.md

```diff
@@ -59,6 +59,42 @@ Code, connected to your Coder workspace for compute, etc.
 1. In VS Code's left-hand nav bar, click **Remote Explorer** and right-click on
    a workspace to connect.
 
+## JetBrains Gateway
+
+Gateway operates in a client-server model, using an SSH connection to the remote host to install
+and start the server.
+
+Setting up Gateway also involves picking a project directory, so if you have not already done so,
+you may wish to open a terminal on your Coder workspace and check out a copy of the project you
+intend to work on.
+
+After installing Gateway on your local system, you may connect to a Coder workspace as follows
+
+1. Open Gateway, make sure "SSH" is selected under "Remote Development"
+2. Click "New Connection"
+3. In the resulting dialog, click the gear icon to the right of "Connection:"
+4. Hit the "+" button to add a new SSH connection
+   1. For the Host, enter `coder.<workspace name>`
+   2. For the Port, enter `22` (this is ignored by Coder)
+   3. For the Username, enter `coder`
+   4. For the Authentication Type, select "OpenSSH config and authentication agent"
+   5. Make sure the checkbox for "Parse config file ~/.ssh/config" is checked.
+   6. Click "Test Connection" to ensure you setting are ok.
+   7. Click "OK"
+5. Select the connection you just added.
+6. Click "Check Connection and Continue"
+7. Select the JetBrains IDE for your project and the project directory
+   1. Use an SSH terminal to your workspace to create a directory or check out code if you haven't
+      already.
+8. Click "Download and Start IDE" to connect.
+
+| Version   | Status      | Notes                                                    |
+|-----------|-------------|----------------------------------------------------------|
+| 2021.3.2  | Working     |                                                          |
+| 2022.1.1  | Working     | Windows clients are unable to connect to Linux workspace |
+| 2022.2 RC | Not working |                                                          |
```

Comment:
do we know why it's not working? Is there a YouTrack issue we can link to?

Reply:
I'm talking through it with JetBrains on their Slack.  I don't know whether they want a YouTrack issue since its still not clear whether its something nonstandard we are doing or a bug on their side.  I'll definitely update this table if there is something productive to share like an issue number.

Reply:
Have one now!